src/selectors/linesInScope.js

```diff
@@ -0,0 +1,41 @@
+import { getOutOfScopeLocations, getSelectedSource } from "../selectors";
+
+import range from "lodash/range";
+import flatMap from "lodash/flatMap";
+import uniq from "lodash/uniq";
+import without from "lodash/without";
+
+function getLinesOutOfScope(outOfScopeLocations: AstLocation[]) {
```

Comment:
`getLinesOutOfScope` sounds like we are taking lines out of a scope, but I think this is getting the out of scope lines. maybe `getOutOfScopeLines` will be clearer here,